django/db/models/expressions.py

```diff
@@ -620,6 +624,8 @@ def as_sql(self, compiler, connection):
 
 
 class RawSQL(Expression):
+    contributes_to_group_by = True
```

Comment:
Honestly I failed to come up of an usage using `RawSQL` in an ORDER BY expression. Could you provide some pointers?